### PR TITLE
Some Maintenance and Cyberiad tweaks, fixes

### PR DESCRIPTION
**Pull request by FreeStylaLT**
I bring the first part in various maintenance tweaks & fixes, along with some convenience-oriented pipe / cable adjustments.

## Tweaks:

Before:
img
After:
img
Moved the atmos pipes to take the shortest path, straight through the hall instead of the arbitrary glass hallway. Left disposal pipe in there, as the hall provides the shortest route for that disposal path.

Before:
img
After:
img
Removes the electrified grilles. Not a single grille in the entirety of the station's maintenance is electrified, except for these. I do not know why they were electrified, all it does is arbitrarily make a random set of windows far more lethal to break than ***literally any other in the entire maintenance***

Before:
img
After:
img

Before:
img
After:
img

Before:
img
After:
img

Before:
img
After:
img
img

Additional pipe tweaks can be found here:
Before:
img
After:
img

Basically, I tried to clean up some of the in-wall nonsense we have. Not all of it, mind you, but it's a small start. Some of it I am willing to let go, like the cryo ones, as there's hardly any reason to ever sabotage or interact with those pipes beyond repairing them.

## Fixes

Fixed everything being duplicated in the surgery room. Someone botched a copy paste job :eyes:

🆑 FreeStylaLT
tweak: Tweaked some pipe / disposal / cable locations in maintenance to not be going through walls
fix: Fixed everything being duplicated in the maintenance surgery room
tweak: Tweaked a couple of maintenance rooms.
/🆑

### CODE REVIEW

**Comment by Lady-Luck** [comment]
>Removes the remnants of previous commits from the Maint PR. Some hints at previous structures is fine, but in cases when they actually existed on the station. As they never actually existed on the station, they shouldn't be there.

I don't agree with this change. I thought it looked really cool, and I actually had a really fun fight around that area earlier today. Please keep it.

**Comment by Norgad** [comment]
>Removes the remnants of previous commits from the Maint PR. Some hints at previous structures is fine, but in cases when they actually existed on the station. As they never actually existed on the station, they shouldn't be there.

That section's theme was 'depricated docking and loading area'. The hints at previous structures are IC'ly

>Removes the electrified grilles. Not a single grille in the entirety of the station's maintenance is electrified, except for these. I do not know why they were electrified

With space on one side, electricity on the other, lockable blast doors, and viewing areas, it's meant to invoke the 'you've fallen right into my trap Mr Bond' feeling

Apart from those parts, good work

**Comment by SamHPurp** [comment]
I agree with all of this. From looking at it, some of the previous PR's images were not updated..

Especially like the scrubber pipe change behind Cryo.

Only thing I noticed @FreeStylaLT is an accidental deletion of wire right of Botany, on this image: https://camo.githubusercontent.com/a4017e4a6060c27436148681e46b3bb0443f6c18/687474703a2f2f692e696d6775722e636f6d2f48424b544f43742e706e67

**Comment by FreeStylaLT** [comment]
That was fixed - the image is outdated

**Comment by Lady-Luck** [comment]
For the record, I submitted a proposal to FreeStyle to keep one of the southern delapitated bridges (the westernmost one) while moving the other one over to north maint in an area that, to be honest, really doesn't ever get used much.

This is a rough Photoshop



The idea is it would give another useful battleground for antags and security in space. Those two little nooks to the left and right of the bridge would be nice little hiding spots security pods can't easily get to. And I think it just looks cool.

**Comment by SamHPurp** [comment]
I hate the 'delapitated bridges'. They look awful. No thanks.

**Comment by FreeStylaLT** [comment]
Undid the tweaks regarding sec maint and bridges. Removed associated images. This PR now only has the pipe and cable modifications.

**Comment by Lady-Luck** [comment]
Why did this get closed?

**Comment by FreeStylaLT** [comment]
This was closed due to #7025 encompassing a good bulk of the pipe changes around Sec, which was roughly half of the PR's contents.
